securedrop/tests/functional/functional_test.py

```diff
@@ -77,6 +79,28 @@ def _unused_port(self):
         s.close()
         return port
 
+    def set_tbb_securitylevel(self, toLevel):
+        if self.torbrowser_driver is None:
+            self.create_torbrowser_driver()
+        driver = self.torbrowser_driver
+
+        driver.get("about:config")
+        accept_risk_button = driver.find_element_by_id("warningButton")
+        if accept_risk_button:
+            accept_risk_button.click()
+        ActionChains(driver).send_keys(Keys.RETURN).\
+            send_keys("extensions.torbutton.security_slider").perform()
+        time.sleep(2)
```

Comment:
this looks great - one thought is that sleeps can be flaky (i.e. on slow machines), can we use `wait_for` here?

Reply:
Added in a couple, but didn't see an easy way to do so on the XUL panel changes from Selenium. Slowness *shouldn't* be an issue as these are local UI updates, not network calls.